### PR TITLE
Update deps and credo

### PR DESCRIPTION
**Pull request by Rados13**

### CODE REVIEW

**Comment by mickel8** [comment]
Hmm, why do we need this?

**Review by daniel-jodlos — APPROVED** [review]

File: LICENSE [hunk 1]
```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2018 Software Mansion
```
Comment (bblaszkow06) [inline LICENSE:189]: The year here is the start of the copyright protection, it should not change

File: lib/membrane_file/source.ex [hunk 2]
```diff
@@ -17,17 +17,23 @@ defmodule Membrane.File.Source do
                 spec: pos_integer(),
                 default: 2048,
                 description: "Size of chunks being read"
+              ],
+              caps: [
+                spec: any(),
+                default: %RemoteStream{type: :bytestream, content_format: nil},
+                description: "Caps which will be sent from Source"
```
Comment (bblaszkow06) [inline lib/membrane_file/source.ex:24]: As discussed, this should not change

**Review by bblaszkow06 — CHANGES_REQUESTED** [review]
We can merge the .credo & mix updates, but not the main part of this PR. You can create a new one with a renamed branch to prevent confusing description of the merge commit

File: lib/membrane_file/source.ex [hunk 3]
```diff
@@ -17,17 +17,23 @@ defmodule Membrane.File.Source do
...
               ]
 
-  def_output_pad :output, caps: {RemoteStream, type: :bytestream}
+  def_output_pad :output, caps: :any
```
Comment (mat-hek) [inline lib/membrane_file/source.ex:27]: whatever is the reason for that, it's most probably not the way

**Review by mat-hek — CHANGES_REQUESTED** [review]

**Review by bblaszkow06 — APPROVED** [review]
The changes look fine, just make sure they are merged in a clean way - squash them, rename the branch and PR title

**Comment by Rados13** [comment]
Changes included in #30.

**Pull request closed without merging**